### PR TITLE
guide: 820c: openembedded

### PR DESCRIPTION
**Pull request by ric96**
Added Guide to flash OE from source for 820c following structure
from 410c

Added a section for 820c in CE/guides/open_embedded.md

Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: ConsumerEdition/dragonboard820c/build/open-embedded.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+be `rpb-console-image`
+
+These will be found in your `tmp-rpb-glibc/deploy/images/dragonboard-820c` directory.
+
+After getting into `fastboot` mode, following commands can be used to flash built images onto Dragonboard820c's UFS.
+
+```shell
+$ gunzip IMAGE-dragonboard-820c.ext4.gz
+$ sudo fastboot flash boot boot-dragonboard-820c.img
+$ sudo fastboot flash rootfs IMAGE-dragonboard-820c.ext4
```
Comment (Mani-Sadhasivam) [inline ConsumerEdition/dragonboard820c/build/open-embedded.md]: There is no `root` partition for 820c. All we have is pure AOSP based partition layout (system and userdata for RFS). I think you can specify `system` here.

@ndechesne Your suggestion?

Comment (ric96) [reply ConsumerEdition/dragonboard820c/build/open-embedded.md]: ok, so i just checked the wiki again seems like debian root is flashed on ```userdata``` and not ```system``` , is it the same for OE?

Comment (Mani-Sadhasivam) [reply ConsumerEdition/dragonboard820c/build/open-embedded.md]: Not sure. IMO `system` partition (naming) makes sense for holding the RFS. But considering the size, `userdata` is better. So, we can specify `system` but a note should be added saying that the user can also use `userdata` partition if more space is needed.

Again I will fall back to what @ndechesne says :)

Comment (sdrobertw) [reply ConsumerEdition/dragonboard820c/build/open-embedded.md]: I will wait for Nico's input.

**Comment by shovanuk** [comment]
Guys update on this?

**Comment by ndechesne** [comment]
hey, sorry.. i missed this one.. way-too-many emails from github..

the default partition to use is set in the OE dragonboard820c configuration file ,at http://git.yoctoproject.org/cgit/cgit.cgi/meta-qcom/tree/conf/machine/dragonboard-820c.conf#n21.

and it is set to system. 

usually on boards that use the android layout, we default to system for OE and userdata for debian. when we use our unified layout with 'rootfs', then we use it by default.

**Comment by sdrobertw** [comment]
@ric96 please apply changes recommended by @Mani-Sadhasivam taking @ndechesne notes into account. Will merge promptly afterwards.

Thanks!

**Comment by ric96** [comment]
@sdrobertw i amended them, please check

**Pull request merged**